contents/teams/crm/objectives.mdx

```diff
@@ -1,3 +1,14 @@
-### Q2 2025 Objectives
+We're currently building on top of PostHog's strengths and transforming group analytics into a powerful customer success platform.
 
-Coming soon.
\ No newline at end of file
+For Q2 2025, our objectives are:
+
+**Increase engagement with group analytics pages**
+* Group and group type overview dashboards.
+* MaxAI summaries of groups.
+* Computed properties for groups.
+* Health score component.
+* Incorporate revenue analytics into the group overview dashboard.
```

Comment:
❤️²

Do you have clearer plans on how to do this? Is this linking to rev analytics OR just exposing them as computed properties?

Reply:
@rafaeelaudibert I haven't looked into it at depth yet, no.

At a minimum, it's the current MRR/ARR as a static value, and a chart displaying MRR/ARR over time. I think I'll do a bit more discovery in advance to understand what customers expect to see.